### DISCUSSION TITLE
Hotkeys for targeting specific application's volume

### DISCUSSION THREAD

**Discussion by StephBoyardee** [File-New-Project/EarTrumpet#1467] [Ideas]
This app is almost perfect, but I don't want to have to move my mouse to change a specific application. I'd love to be able to have 3 or 4 extra hotkeys. Two application specific volume hotkeys that will turn the targeted application volume up or down, and 1 or 2 to cycle through/target applications for specific volume changes. I could foresee the UI being close to the mixer window, except it's just a single icon and volume bar, with two arrows next to the icon indicating that it can change. If anyone wants to point me in the direction of specific classes or functions to look at that would lend to creating this, I wouldn't mind tackling it myself if people are interested. 

Cheers

**Comment by riverar** [comment]
Hey @StephBoyardee, dev builds (see https://eartrumpet.app) have EarTrumpet Actions which should let you do what you need. Give it a spin and let us know!

**Reply by StephBoyardee** [reply]
It seems that I can make a set of volume buttons for each application, but it doesn’t seem like cycling through the applications is possible.  Thanks for pointing me to the dev version tho!
